### ISSUE TITLE
Why do I see that the Real-ESRGAN folder is empty? 

### ISSUE THREAD

**Issue by khoitadev** [saifhassan/Wav2Lip-HD#21]
Why do I see that the Real-ESRGAN folder is empty? What should I add to this folder? If using git clone https://github.com/xinntao/Real-ESRGAN.git, the path Real-ESRGAN/gfpgan/weights/ does not exist. I hope to receive help from everyone

**Comment by saifhassan** [comment]
> Why do I see that the Real-ESRGAN folder is empty? What should I add to this folder? If using git clone https://github.com/xinntao/Real-ESRGAN.git, the path Real-ESRGAN/gfpgan/weights/ does not exist. I hope to receive help from everyone

yeah, you can clone that repo and download weights from [ESRGAN](https://github.com/saifhassan/Wav2Lip-HD#testing-model).

**Comment by khoitadev** [comment]
Sorry but I don't fully understand your answer, can you help me explain it more clearly? What I'm understanding here is
step 1: git clone https://github.com/xinntao/Real-ESRGAN.git
step 2: cd Real-ESRGAN, then download gfpgan folder from git repository https://github.com/TencentARC/GFPGAN.git

I hope you can explain each step more clearly. I sincerely thank you for your help

**Comment by JuvenileLocksmith** [comment]
> > Why do I see that the Real-ESRGAN folder is empty? What should I add to this folder? If using git clone https://github.com/xinntao/Real-ESRGAN.git, the path Real-ESRGAN/gfpgan/weights/ does not exist. I hope to receive help from everyone
> 
> yeah, you can clone that repo and download weights from [ESRGAN](https://github.com/saifhassan/Wav2Lip-HD#testing-model).

even after you clone the repo down, it still will not work. Been there, done that.

**Comment by saifhassan** [comment]
> Sorry but I don't fully understand your answer, can you help me explain it more clearly? What I'm understanding here is step 1: git clone https://github.com/xinntao/Real-ESRGAN.git step 2: cd Real-ESRGAN, then download gfpgan folder from git repository https://github.com/TencentARC/GFPGAN.git
> 
> I hope you can explain each step more clearly. I sincerely thank you for your help

yeah., these steps are fine. These should work.

**Comment by AIhasArrived** [comment]
Did you guys have to change the code?
Is this how your folder should look like ?? @JuvenileLocksmith  @khoitadev  @saifhassan 
![image](https://github.com/saifhassan/Wav2Lip-HD/assets/148225825/d6bb371d-f678-4dca-a379-8273ccc75926)

and inside it:
![image](https://github.com/saifhassan/Wav2Lip-HD/assets/148225825/b9f5c277-1e49-4c94-8083-62ac5d04f0f2)

I still get this error:

```
Real-ESRGAN\inference_realesrgan.py", line 119, in main
    from gfpgan import GFPGANer
ImportError: cannot import name 'GFPGANer' from 'gfpgan' (unknown location
```

I went checking the code it says:
```
if args.face_enhance:  # Use GFPGAN for face enhancement
        from gfpgan import GFPGANer
        face_enhancer = GFPGANer(
            model_path='https://github.com/TencentARC/GFPGAN/releases/download/v1.3.0/GFPGANv1.3.pth',
            upscale=args.outscale,
            arch='clean',
            channel_multiplier=2,
            bg_upsampler=upsampler)
```

Isn't there a problem with this code maybe???
